src/protocol/helper.c

```diff
@@ -492,18 +511,18 @@ protocolRemoteGet(ProtocolStorageType protocolStorageType, unsigned int hostIdx)
     {
         MEM_CONTEXT_BEGIN(protocolHelper.memContext)
         {
-            unsigned int optHost = isRepo ? cfgOptRepoHost : cfgOptPgHost + hostIdx;
+            unsigned int optHost = isRepo ? cfgOptRepoHost : cfgOptPgHost;
```

Comment:
I still feel like we're putting the cart before the horse with hostIdx - it seems we are assuming the hostIdx will be the same for pg and for repo since on line 519 the host could be repo but it could be pg. This commit does not deal with multi repo yet, but we are treating the hostIdx as if it applies to the repo stuff as well.